app/components/primer/component.rb

```diff
@@ -55,6 +55,13 @@ def validate_aria_label
       raise ArgumentError, "`aria-label` is required." if aria_label.nil? && !Rails.env.production?
     end
 
+    def validate_aria_labeled
+      return if Rails.env.production?
+
+      aria_label = aria("label", @system_arguments) || aria("labelledby", @system_arguments) || aria("describedby", @system_arguments)
```

Comment:
A label is always required, so we should check for `aria-label` or `aria-labelledby`. I see we're already checking for `aria-label` on line 55, so maybe we just move the label/labelledby checks to that block?

`aria-describedby` is only required when a caption has been passed.